lib/models/addon.js

```diff
@@ -332,6 +332,7 @@ Addon.prototype.compileStyles = function(tree) {
   if (tree) {
     var self = this;
     var processedStyles = preprocessCss(tree, '/', '/', {
+      outputPaths: { 'addon': this.name + '.css' },
```

Comment:
the funnel below this that looks for app.css and renames it to this.name is unneeded